backend/model/wrapper.js

```diff
@@ -200,13 +222,14 @@ function getTeams(connection, tournamentId) {
   return teamWrapper.getTeams(connection, tournamentId);
 }
 
-function updateTeam(connection, id, teamName, leader, tournament, seed) {
+function updateTeam(connection, id, teamName, leader, tournament, paid, seed) {
```

Comment:
It's not the job of the wrapper to really differentiate between who calls the functions or what is passed to the functions, this will need to be monitored by the backend and frontend; this function exists so that team can be updated in the ways that are possible. A counterpoint either way is that a team's paid status may be updated by a referee upon payment of cash in person, for which case this endpoint is useful; however, if truly needed separately the payment status can be set using the existing updateTeamField function.